qt/python/mantidqt/widgets/fitpropertybrowser/addfunctiondialog.py

```diff
@@ -0,0 +1,45 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+from __future__ import (absolute_import, unicode_literals)
+
+from qtpy.QtCore import Qt, Signal
+from qtpy.QtGui import QIcon
+from qtpy.QtWidgets import QCompleter, QDialog
+
+from mantidqt.utils.qt import load_ui
+
+
+class AddFunctionDialog(QDialog):
```

Comment:
Would it be possible to add some tests around this? Ideally the MVP style would help make this more testable but a test using `@start_qapplication` (https://github.com/mantidproject/mantid/blob/master/qt/python/mantidqt/utils/qt/testing/__init__.py#L22) would allow this to be constructed and at least the `action_add_function` method tested.